gcp-cloud-run/build.gradle

```diff
@@ -21,7 +21,7 @@ bootRun {
 }
 
 bootJar {
-    mainClassName = 'ai.h2o.mojos.deploy.gcp.cloud.run.GcpScorerApplication'
+    mainClass = 'ai.h2o.mojos.deploy.gcp.cloud.run.GcpScorerApplication'
```

Comment:
qq - what does the renaming have to do with the version change? Is it required?

Reply:
it got deprecated in new version https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.4-Release-Notes#spring-boot-gradle-plugin